controller/network/network.go

```diff
@@ -253,8 +253,9 @@ func (network *Network) ConnectRouter(r *Router) {
 	network.routerChanged <- r
 
 	for _, h := range network.routerPresenceHandlers {
-		h.RouterConnected(r)
+		go h.RouterConnected(r)
```

Comment:
This is an interesting little change. In other event callbacks, we have chosen not to put them in go routines and instead let the callback logic decide.

I am not advocating for change. I am highlighting it.

Reply:
My thinking was that because slowdowns here can delay accepting control channels, we should handle this asynchronously.